app/uk/gov/hmrc/plasticpackagingtax/returns/models/domain/HumanMedicinesPlasticWeight.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.plasticpackagingtax.returns.models.domain
+
+import play.api.libs.json.{Json, OFormat}
+
+case class HumanMedicinesPlasticWeight(totalKg: Option[Long] = None)
```

Comment:
Yes this is the thing I don't like. But I am addressing on my ticket. Happy to continue for now, but if something is mandatory we should not mark it as optional.

I will discuss with you later :)

Reply:
ok, cool, happy to discuss, I'll merge that for now, and we can catch up tomorrow on the way forward